### ISSUE TITLE
Monitor off crashes alacritty/waybar/firefox

### ISSUE THREAD

**Issue by Kommynct** [hyprwm/Hyprland#6437]
### Bug or Regression?

Bug

### System Info and Version

<details> <summary>System/Version info</summary>

```sh
Hyprland, built from branch  at commit ea2501d4556f84d3de86a4ae2f4b22a474555b9f  (props: bump version to 0.41.0).
Date: Mon Jun 10 14:23:06 2024
Tag: v0.41.0, commits: 4794

flags: (if any)


System Information:
System name: Linux
Node name: anarchy-desktop
Release: 6.9.3-zen1-1-zen
Version: #1 ZEN SMP PREEMPT_DYNAMIC Fri, 31 May 2024 15:14:26 +0000


GPU information: 
06:00.0 VGA compatible controller [0300]: Advanced Micro Devices, Inc. [AMD/ATI] Navi 23 [Radeon RX 6650 XT / 6700S / 6800S] [1002:73ef] (rev c1) (prog-if 00 [VGA controller])


os-release: NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
BUILD_ID=rolling
ANSI_COLOR="38;2;23;147;209"
HOME_URL="https://archlinux.org/"
DOCUMENTATION_URL="https://wiki.archlinux.org/"
SUPPORT_URL="https://bbs.archlinux.org/"
BUG_REPORT_URL="https://gitlab.archlinux.org/groups/archlinux/-/issues"
PRIVACY_POLICY_URL="https://terms.archlinux.org/docs/privacy-policy/"
LOGO=archlinux-logo


plugins:

```

</details>

### Description

alacritty crashes

### How to reproduce

Disable live updates, run hyprctl reload in alacritty, find that it crashes alacritty.

### Crash reports, logs, images, videos

[hyprlandCrashReport784.txt](https://github.com/user-attachments/files/15798823/hyprlandCrashReport784.txt)
[hyprlandCrashReport384151.txt](https://github.com/user-attachments/files/15798824/hyprlandCrashReport384151.txt)
[hyprlandCrashReport789.txt](https://github.com/user-attachments/files/15798825/hyprlandCrashReport789.txt)
[hyprlandCrashReport764.txt](https://github.com/user-attachments/files/15798826/hyprlandCrashReport764.txt)

**Comment by Kommynct** [comment]
I've realized that it only crashes if my TV is turned on, which is off by default

```
monitor=HDMI-A-1,disabled,highrr,0x0,2,bitdepth,10
monitor=DP-1,highrr,1280x0,1,bitdepth,10
monitor=DP-2,preferred,3840x0,1,bitdepth,10

bind = SUPER, c, exec, hyprctl keyword monitor 'HDMI-A-1,highrr,0x0,2,bitdepth,10' ; kanshictl switch TV ; pactl set-card-profile alsa_card.pci-0000_06_00 output:hdmi-stereo && pactl set-default-sink alsa_output.pci-0000_06_00.hdmi-stereo & hyprctl dispatch closewindow mpvrainsounds & swaymsg input '117:256:Paris_Commune_Mouse' map_to_output 'HDMI-A-1' & ~/.config/tofi/scripts/smartthings devices:commands 1526283b-2f10-4f44-8eb1-d7226b49ece7 switch:on || notify-send --urgency=low "TV is offline" && sleep 2 && "kanshictl switch TV & hyprctl keyword monitor 'HDMI-A-1,highrr,0x0,2,bitdepth,10'"

```

it crashes when the TV turns off when hyprctl reload happens and runs the monitor disabled.
[log.txt](https://github.com/user-attachments/files/15812738/log.txt)

**Comment by Kommynct** [comment]
It also seems to crash firefox and waybar sometimes.

**Comment by khaneliman** [comment]
Alacritty crash on monitor off and on: https://gist.github.com/khaneliman/899f93619f21cded0cb34765394d6aa0

**Comment by Kommynct** [comment]
fixed by 0.42.0 for me
